### PR TITLE
please review and suggest necessary changes

### PR DESCRIPTION
**Pull request by naman99-agar**

### CODE REVIEW

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@naman99-agar Changes Required!
1. Rename "Images and Video" as "Images and Videos". Similarly, Rename Video as "Videos".
2. Move CAD images into Mechanical Design. You can have full view CAD Images inside Images folder if you don't have other project related images.
3. Justify text in Report.
4. In README.md, remove heading "Images" and have only one picture at the top.
5. In README.md, Arduino code has to be a sub-heading under main heading "Electronics Aspect of Bot"
6. In README.md, move workflow above Mechanical Aspcet.
7. In README.md, hyperlink team members and mentors names with their GitHub profile link.
8. In README.md, justify text using HTML tags.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,110 @@
...
+1.  The bot cannot exactly predict the trajectory of the puck. And therefore it is unable to hit the puck in a particular direction.
+2.  It is not competent enough to detemine the velocity of the puck so as to hit it with a specific force.
+3.  No plans regarding the introduction of the difficulty level option is executed till now.
+ 
+## Future improvements 
+1.  LCD display module can be installed to interact with the bot like setting up difficulty level, counting scores etc.
+2.  Image processing can be incorporated to contemplate trajectories.
+ 
+## Team members
+1. Mohit Singh (https://github.com/mohitschoudhary)
```
Comment (agshubhs) [inline README.md]: @naman99-agar 
Syntax is wrong it should be `` [Mohit Singh](https://github.com/mohitschoudhary)``  [Mohit Singh] which gives (https://github.com/mohitschoudhary)
Do this for all the names

**Comment by naman99-agar** [comment]
the changes are done, please review

File: README.md [hunk 2]
```diff
@@ -1,2 +1,110 @@
-# Air-Hockey-Robot
-Srishti 2020
+# AIR HOCKEY ROBOT
+### Shrishti'20, IITR
+## Image of the project
```
Comment (SanjeevKrishnan) [inline README.md]: Remove this sub-heading and remove '1.' at the beginning from "1. Table isometric view"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,110 @@
...
+1. Table isometric view 
+![Table isometric view 1](https://user-images.githubusercontent.com/68807083/88667192-a8c56200-d0fe-11ea-8249-519fb2419034.jpg)
+
+## Abstract
+<p align="justify">To create an autonomous gaming system of air hockey where one player is human and other side will be played by a bot. The bot will automatically detect the position of the moving puck at every instant and defend or strike accordingly. The aim is to set up difficulty levels as well whose control will be subject to the human player.
```
Comment (SanjeevKrishnan) [inline README.md]: When you start a paragraph with `<p>` tag you should end it with `<\p>`. For example `<p align="justify">Sample Text<\p>`. Do this change throughout README.md

File: README.md [hunk 4]
```diff
@@ -1,2 +1,110 @@
...
+## Team members
+1. [Mohit Singh](https://github.com/mohitschoudhary)
+2. [Naman Agarwalla](https://github.com/naman99-agar)
+3. [Naman Pesricha](https://github.com/pesricha)
+4. [Ojus Thool](https://github.com/ojusthool)
+5. [Pranav Kumar](https://github.com/Pranav123-alt)
+6. [Rishita](https://github.com/Rishita-003)
+
+## Mentors 
+1. Tabish Madni (https://github.com/Tmadni)
```
Comment (SanjeevKrishnan) [inline README.md]: Do the same syntax change in Mentors list too which you did for Team Members

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@naman99-agar Minor Changes

**Comment by naman99-agar** [comment]
please review

**Review by SanjeevKrishnan — APPROVED** [review]
@naman99-agar PR looks fine!
@Shubhanshu07 I suggest for merge!

**Comment by agshubhs** [comment]
@Great work PR looks good to  me

**Pull request merged**